infra/ansible/roles/build_srcs/tasks/main.yaml

```diff
@@ -1,3 +1,18 @@
+- name: Read PyTorch pin
+  ansible.builtin.command: cat {{ (src_root, 'pytorch/xla/.torch_pin') | path_join }}
```

Comment:
There are actually 2 places historylly where we put `.torch_pin`, one is `pytorch/xla/.torch_pin`, another one is `pytorch/xla/torch_patches/.torch_pin`. I think it might be better to deprecate the `pytorch/xla/.torch_pin` since we only use this for the release branches.

Reply:
Yeah, I did see half of the references in this repo were to one place, and half were to the other place. I weakly prefer just deleting the `torch_patches` directly since (as far as I know) we never add patches and our build scripts do not apply them.

Reply:
Discussed offline. Moving this under `.github` since this mainly exists for CI anyway. Updated all of the docs too.